### DISCUSSION TITLE
Should we release 0.71.13?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#78] [Patches]
## Released, please refer to #87

---

Current Release: [0.71.12](https://github.com/facebook/react-native/releases/tag/v0.71.12)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Picked

1. https://github.com/facebook/react-native/commit/39016889d09f1766e218e52d2e141e960a35b38a
1. https://github.com/facebook/react-native/commit/f8c03d18eddcce5228448f68898fbcf17e1881c9
1. https://github.com/facebook/react-native/commit/91578418f91d845d30aefc0be766810d18936530
1. https://github.com/facebook/react-native/commit/0c9c57a9f73294414d92428c5d2472dc1e1e5e96
1. https://github.com/facebook/react-native/commit/177d97d8ea962bdd4dad8fcf0efb04a307f25000

#### Local commits to backport to main

1.

**Comment by Saadnajmi** [comment]
Pick request: https://github.com/facebook/react-native/commit/f7219ec02d71d2f0f6c71af4d5c3d4850a898fd8
Helps with JSC debugging

**Reply by fortmarek** [reply]
@Saadnajmi this cherry-pick causes merge conflicts. Do you think you can prepare a PR for this to `0.71-stable`?

**Reply by Saadnajmi** [reply]
Draft here: https://github.com/facebook/react-native/pull/38995

I also ran into some merge conflicts, so I want to do some testing before marking this as ready.

**Reply by Saadnajmi** [reply]
Should be ready now! I'm also going to port to 0.72 as I see I need to do that...

**Reply by fortmarek** [reply]
Picked 🤝

**Comment by dominikwagner** [comment]
Would it be possible to include the PR [38256](https://github.com/facebook/react-native/pull/38256) in this release? Because at the moment this is blocking Android 14 support.

**Reply by JMalone3** [reply]
Pick request: [facebook/react-native@177d97d](https://github.com/facebook/react-native/commit/177d97d8ea962bdd4dad8fcf0efb04a307f25000)
Blocking Android 14 support. Please escalate. Our project has an OEM deliverable due early September with RN 0.71.

**Reply by fortmarek** [reply]
We plan to release 0.71 and 0.72 soon (ETA this or next week). Sorry for the wait!

**Comment by kelset** [comment]
just for reference to the release crew, this is the high-prior issue for MSFT that I mentioned at the last weekly sync that is getting looked into -  once it's addressed we'd want to backport to 71 https://github.com/facebook/react-native/issues/38024 (and 72 most likely)

**Reply by fortmarek** [reply]
Commit: https://github.com/facebook/react-native/commit/39016889d09f1766e218e52d2e141e960a35b38a

However, there's a backport [PR](https://github.com/facebook/react-native/pull/38784), we should cherry-pick instead.

**Reply by shivenmian** [reply]
+1 to this - now that the backport PR has been merged, would really appreciate this being picked in 0.71.13. cc @rasaha91

**Reply by fortmarek** [reply]
It's now picked and will be part of the next patch release

**Comment by kelset** [comment]
release crew specific improvement to merge asap (after patch is ok) -> https://github.com/facebook/react-native/pull/38890

doesn't need a new patch, but merge for consistency across all versions in support window + main

**Reply by fortmarek** [reply]
Picked 🤝

**Comment by douglasjunior** [comment]
Pick request: https://github.com/facebook/react-native/commit/0c9c57a9f73294414d92428c5d2472dc1e1e5e96

Fix onChangeText not firing when clearing the value of TextInput with multiline=true on iOS

**Reply by douglasjunior** [reply]
@kelset please, how do I know if this other commit was merged before?

https://github.com/facebook/react-native/commit/a804c0f22b4b11b3d9632dc59a6da14f6c4325e3

Perhaps the two are complementary.

I've documented the multiline behavior between versions in this comment: https://github.com/facebook/react-native/issues/36494#issuecomment-1678773893

**Reply by kelset** [reply]
when cherry-picking a commit you make an exact replica in the new branch, so the only way to figure it out is actually checking if the commit message appears in that git log of that branch. I usually do it via the GH interface for commit history, or via a git GUI client (fork in my case)

**Reply by fortmarek** [reply]
The other commit (https://github.com/facebook/react-native/commit/a804c0f22b4b11b3d9632dc59a6da14f6c4325e3) seems to be already in 0.71, so we won't pick that.

We will pick the initial request.

**Comment by fortmarek** [comment]
Cherry-pick maven ping release improvement: https://github.com/facebook/react-native/pull/38980

**Reply by fortmarek** [reply]
Picked 🤝

**Comment by AlexanderEggers** [comment]
Would it be possible to pick this commit? https://github.com/facebook/react-native/commit/133ccdcc67a7d19ffa5130949893c2792e3ad9fb

It would be great to get this small one into the next 0.71 patch release. It resolves an issue around the useEffect regression introduced in 0.69.

**Reply by edstbbz** [reply]
I would also like to see it in this release

**Reply by sunnylqm** [reply]
https://github.com/reactwg/react-native-releases/discussions/69#discussioncomment-6048229

**Reply by fortmarek** [reply]
We plan to release 0.71 soon, so this won't make it in `0.71.13`. @cortinico any plans to backport this eventually to 0.71?

**Reply by cortinico** [reply]
Nope we're not planning on backporting this change to older versions. This is a really invasive fix and we feel that backporting it in a patch version of React Native 0.71 could create more harm than good.

**Comment by Saadnajmi** [comment]
Can we pick https://github.com/facebook/react-native/pull/37914 and https://github.com/facebook/react-native/pull/39037 ? The former allows direct debugging on JSC with iOS 16.4, while the latter makes the change backwards compatible to earlier versions of Xcode :)

**Reply by fortmarek** [reply]
We've already tested the current patch, so this will have to come in the subsequent one 😞

**Reply by Saadnajmi** [reply]
> We've already tested the current patch, so this will have to come in the subsequent one 😞 

No worries, thanks for the update!
